File: Kritika Tripathi/Basics Of Programming/Patterns/Pattern11.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    int count=1;
    //write your code here
    for(int i=1; i<n; i++){
        for(int j=1; j<=i; j++, count++){
            cout<<count<<"\t";
        }
        cout<<endl;
    }
    
}

File: V.Vennela/Basics of Programming/Patterns/Pattern11.java
import java.util.*;

public class Pattern11 {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        // write ur code here
        int n = scn.nextInt();
        int c=1;
        for(int i=1;i<=n;i++)
        {
            for(int j=1;j<=n;j++)
            {
                if(j<=i)
                {
                    System.out.print(c+"\t");
                c++;
                }
                else
                    System.out.print("\t");
                
            }
            System.out.println();
        }

    }
}
